### ISSUE TITLE
Example of a Relaton mini-site

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/relaton.com#3]
A Relaton/Metanorma mini-site is useful for creating a simple document registry where the documents are all Metanorma files.

Here's an example:
https://riboseinc.github.io/csd-public-documents/

We should describe and offer this as a solution based on Relaton for people who need this.

**Comment by strogonoff** [comment]
I believe this is primarily about https://github.com/relaton/relaton-server-csd?

**Comment by ronaldtse** [comment]
@strogonoff yes, and standards.calconnect.org has not been generalized yet. Maybe you can help make it a Jekyll plugin?

**Comment by ronaldtse** [comment]
We now have generalized standards.calconnect.org.
